Add subsection on edge prior results (task 2/3) (#25)
greenelab/xswap-manuscript: content/02.body.md

![Calibration figure](){#fig:calibration width="100%"}

The second prediction task mirrored the first task, but it involved reconstructing networks based on subsampled networks with only 70% of the original edges.
Because edges were sampled uniformly without replacement, the subsampled networks share similar degree distributions to the original networks (see Figure {@fig:degree}B).
Unlike in the first task, edges that were present in the sampled network were not tested and therefore are not included in the performance metrics.
The results of the second prediction task further demonstrate a high level of performance for degree-sequence-based node pair features (Figure {@fig:discrimination}).
The edge prior was able to reconstruct the unsampled network with an AUROC of greater than 0.9 in 14 of 20 networks.
As was observed in the first task, node pair features computed in second prediction task were highly rank-correlated, meaning the AUROC values for different features were similar.

> While reconstruction performance was, as expected, slightly lower in the second task compared to the first, we found that many networks were still able to be reconstructed with a high level of performance.

Reviewer: ```suggestion
While performance was slightly lower in the second task than the first, we found that many networks were still well reconstructed.
```

> The edge prior was the best calibrated feature in both the first and second prediction tasks.

Reviewer: ```suggestion
The edge prior was the best calibrated feature for both tasks.
```
